src/app/core/components/left-nav-tree/left-nav-tree.component.html

```diff
@@ -116,7 +123,7 @@
     <span
       id="nav-{{ node.data.element.id }}"
       class="node-tree-item folder"
-      [ngClass]="{ expanded: node.expanded, locked: node.locked, current: node.data && node.data.current }"
+      [ngClass]="{ expanded: node.expanded, locked: node.locked, current: node.data && node.data.current, disabled: node.data.element.disabled }"
```

Comment:
"locked" seems not to be used anymore, probably it was done for this. 
I think I would prefer you reuse the "locked" class for this. 

And `node.data.element` is a `NavTreeElement`, it does not have any "disabled". It only works because the angular templating forgets about the type and makes everything 'any'.
So just introduce a new `locked` property in `NavTreeElement` and make sure it has a value for items and groups. (it may be made more clean later)

Reply:
Fixed